src/components/AddCardLink.js

```diff
@@ -1,4 +1,4 @@
 import React from 'react'
 import {AddCardLink} from 'rt/styles/Base'
 
-export default ({onClick, t}) => <AddCardLink onClick={onClick}>{t('Click to add card')}</AddCardLink>
+export default ({onClick, t, laneId}) => <AddCardLink onClick={onClick}>{t('Click to add card')}</AddCardLink>
```

Comment:
Do we really need this argument? We don't use it in this component

Reply:
Yes. We do not use them in this component. But when we use custom components, there arises a use-case where we need the laneId, and since we are destructuring here, we wouldn't have access to the props right. That's why I included it here.

Reply:
I mean the `laneId` is still unused parameter after destructuring. But, I am going to merge this for now.